src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -44,7 +44,7 @@
 
 vsts_account_name = CliArgumentType(
     options_list=('--vsts-account-name',),
-    help='VSTS account name to create the build and release definitions. A new VSTS account is created if omitted or does not exist.'
+    help='VSTS account name to create the build and release definitions. If you have an existing VSTS account at myvstsaccount.visualstudio.com, then set this parameter value to myvstsaccount. A new VSTS account is created if parameter value is omitted or does not exist.'
 )
 
 vsts_project_name = CliArgumentType(
```

Comment:
@tjprescott Isn't the approach you just mentioned equivalent to what is already there? My main concern was with commands not using the registered arguments. If they are ignored I can switch to just having 'container' for now.